include/seqan3/alignment/pairwise/detail/policy_alignment_matrix.hpp

```diff
@@ -138,14 +142,15 @@ class policy_alignment_matrix
         size_t const column_count = sequence1_size + 1;
         size_t row_count = sequence2_size + 1;
 
+        alignment_matrix.resize(column_index_type{column_count}, row_index_type{row_count}, initial_score);
+
```

Comment:
```suggestion
```

Why do you need this twice? :joy:

Reply:
It was supposed to be the index matrix :)